.dev/config/geoschem_config.yml

```diff
@@ -89,7 +89,7 @@ operations:
   photolysis:
     activate: true
     input_directories:
-      fastjx_input_dir: /home/joe/data/GISS-GC/prod_input_files/ExtData/CHEM_INPUTS/FAST_JX/v2021-10/
```

Comment:
`FAST_JX` input data updated to more recent version, as recommended by @ltmurray.